internal/hmyapi/types.go

```diff
@@ -41,6 +44,35 @@ type RPCCXReceipt struct {
 	Amount      *hexutil.Big    `json:"value"`
 }
 
+// HeaderInformation represents the latest consensus information
+type HeaderInformation struct {
+	BlockHash   common.Hash `json:"blockHash"`
+	BlockNumber uint64      `json:"blockNumber"`
+	ShardID     uint32      `json:"shardID"`
+	Leader      string      `json:"leader"`
+	ViewID      uint64      `json:"viewID"`
+	Epoch       uint64      `json:"epoch"`
+	Timestamp   string      `json:"timestamp"`
+	UnixTime    uint64      `json:"unixtime"`
```

Comment:
also return last sig and bitmap?

Reply:
since it's the latest block, we don't have the corresponding sig and bitmap. if directly read from ReadLastCommits, it might not be synchronized in some cases

Reply:
but we should just return the sig as is, right? Just return the sig in this header itself.

Reply:
yes, that should be easier.